generators/app/templates/package.json

```diff
@@ -4,8 +4,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.12",
     "@fortawesome/free-solid-svg-icons": "^5.6.3",
     "@fortawesome/react-fontawesome": "^0.1.4",
-    "@inrupt/inrupt-atomic-style-guide": "^0.1.13",
-    "@inrupt/solid-react-components": "^0.1.0",
+    "@inrupt/inrupt-atomic-style-guide": "^0.1.14",
+    "@inrupt/solid-react-components": "^0.2.0-rc.0",
```

Comment:
Marking this as a requested change so it blocks the PR until we decide what to do with the versioning and releases